cloud/scope/clients.go

```diff
@@ -192,19 +192,6 @@ func (c *ClientProvider) createVcnClient(region string, ociAuthConfigProvider co
 	dispatcher := vcnClient.HTTPClient
 	vcnClient.HTTPClient = metrics.NewHttpRequestDispatcherWrapper(dispatcher, region)
 
-	if c.certOverride != nil {
-		if client, ok := dispatcher.(*http.Client); ok {
-			err = c.setCerts(client)
-			if err != nil {
-				logger.Error(err, "unable to create OCI VCN Client")
-				return nil, err
-			}
-		} else {
-			return nil, errors.New("The VCN Client dispatcher is not of http.Client type. Can not patch the tls config.")
-		}
-
-	}
```

Comment:
And this is removed since we can now set the certs via an ENV?

Reply:
yep https://github.com/oracle/oci-go-sdk/blob/master/common/tls_config_provider.go#L17

Reply:
Also, we have added volumes in the e2e template? We are missing validation there though. I can add it as separate PR.